events/bingo/serverScripts/bingoScannerZone/bingoScannerZoneServer.js

```diff
@@ -24,7 +24,23 @@
     var WAIT_TO_CLOSE_TRAP_DOOR_MS = 1500;
     var WAIT_TO_CLOSE_WIN_GATE_MS = 2000;
     var WAIT_FOR_ENTITIES_TO_LOAD_MS = 2000;
-    var SPREADSHEET_URL = Script.require(Script.resolvePath('../../secrets/bingoSheetURL.json')).sheetURL;
+    var REQUEST_URL = Script.require(Script.resolvePath('../../secrets/secrets.json?0')).requestURL;
```

Comment:
i see ?0 a couple of times, can that be removed or necessary?

Reply:
necessary in some cases and doesn't hurt anything in others